grails-bom/plugins.properties

```diff
@@ -1,9 +1,10 @@
 cache=8.0.0-SNAPSHOT
+database-migration=6.0.0-SNAPSHOT
```

Comment:
I believe this can be removed, as it was never present in grails-bom.

Reply:
> The few comments, plus I'd like to add these:
> org.gebish:geb-spock:7.0
> com.bertramlabs.plugins:asset-pipeline-grails:5.0.1
> org.grails.plugins:database-migration:6.0.0-SNAPSHOT

@jamesfredley  It was just added because you asked to add it.

Reply:
Sorry, I thought that was a leftover from before these changes.